group_vars/all/all.yml

```diff
@@ -29,6 +29,7 @@ jenkins_aws_account_id: "791438942407"
 # Environment type - local is for Minikube, aws for AWS :)
 # by default it's local
 fjelltopp_env_type: "local"
+fjelltopp_cloud_provider: ''
```

Comment:
We want to minimise the number of if statements as far as possible, so adding in another switch variable should be avoided as far as possible.  In this case I think it is acceptable as a temporary measure until Craig resolves the problem of the volume naming.